app/src/main/java/org/fossasia/openevent/app/common/app/binding/BindingAdapters.java

```diff
@@ -27,7 +27,11 @@
 import org.fossasia.openevent.app.common.utils.ui.CircleTransform;
 import org.fossasia.openevent.app.module.main.MainActivity;
 
-public class BindingAdapters {
+public final class BindingAdapters {
+
+    private BindingAdapters() {
```

Comment:
https://stackoverflow.com/questions/14398747/hide-utility-class-constructor-utility-classes-should-not-have-a-public-or-def

https://stackoverflow.com/questions/32375149/is-it-mandatory-utility-class-should-be-final-and-private-constructor